### DISCUSSION TITLE
Modify channel informations

### DISCUSSION THREAD

**Discussion by yschulzy** [TwitchIO/TwitchIO#301] [Questions] [answered]
Hello :D
I have a question about Twitchiio. I am in the process of developing a twitch bot for myself and came across your repository. Is it possible with the package to change the stream information like the title or the category ? and if so do you have a reference to an example where I can see how to do it ?

**Comment by chillymosh** [comment, marked as answer]
Hi, yes it is possible using the following method https://twitchio.dev/en/latest/reference.html#twitchio.PartialUser.modify_stream
